backend/src/main/java/com/jobwait/persistence/answerpersistence/LocalDateAnswer.java

```diff
@@ -2,30 +2,29 @@
 
 import java.sql.PreparedStatement;
 import java.sql.SQLException;
-import java.sql.Timestamp;
 import java.sql.Types;
-import java.time.OffsetDateTime;
+import java.time.LocalDate;
 
 import com.jobwait.domain.Answer;
 
-public class OffsetDateTimeAnswer implements Answer<OffsetDateTime> {
-    private OffsetDateTime value;
-    private final int sqlType = Types.TIMESTAMP_WITH_TIMEZONE;
+public class LocalDateAnswer implements Answer<LocalDate> {
+    private LocalDate value;
+    private final int sqlType = Types.DATE;
```

Comment:
Possible concern about timezones not being handled correctly, but I guess May 12 is May 12 regardless of the timezone, so we won't have to convert dates to local time for the user, so I guess this is fine